inyoka_theme_default/templates/macros.html

```diff
@@ -112,7 +112,7 @@
 
 {# FORM #}
 
-{% macro outer_form(csrf, form=None, bootstrap_class=None, action="", method="post", submit_label=None, manually_rendered=False) %}
+{% macro outer_form(csrf, form=None, bootstrap_class=None, action="", method="post", submit_label=None, button_visible=True, manually_rendered=False) %}
```

Comment:
You hide the button and not only the label. So i think that ``button_visible`` is the better name.